docker/quickstart/docker-compose-without-neo4j.quickstart.yml

```diff
@@ -89,8 +89,6 @@ services:
   elasticsearch:
     container_name: elasticsearch
     environment:
-    - discovery.type=single-node
```

Comment:
seems like there is something wrong with the script. Can you make suer the python script that it's running under the hood correctly applies the env along with the variables in the env_file?

Reply:
Can you regenerate these ymls? These two lines should not get deleted

Reply:
Done